test cases/common/175 get project license/meson.build

```diff
@@ -0,0 +1,5 @@
+project('bar', 'c', license: 'Apache')
+
+executable('bar', 'bar.c')
+
```

Comment:
I would make this an error.

if not (  meson.project_license()[0] == 'Apache' )
    error('The license should be Apache, but it is:  ' + meson.project_license()[0] )
endif